deeds-dapp-webapp/src/main/webapp/WEB-INF/metadata/whitepaper.html

```diff
@@ -2,8 +2,8 @@
 <meta name="description" content="#{metadata.whitepaper.pageDescription}">
 <link rel="canonical" href="https://www.meeds.io${servletPath}">
 <meta property="og:locale" content="${lang}">
-<link rel="alternate" href="https://www.meeds.io/fr/livre-blanc" hreflang="fr"/>
-<link rel="alternate" href="https://www.meeds.io/whitepaper" hreflang="en"/>
+<link rel="alternate" href="https://www.meeds.io/dapp/fr/livre-blanc" hreflang="fr"/>
+<link rel="alternate" href="https://www.meeds.io/dapp/whitepaper" hreflang="en"/>
```

Comment:
Will need to add further alternative urls instead of replacing existing ones in fact.

```suggestion
<link rel="alternate" href="https://www.meeds.io/fr/livre-blanc" hreflang="fr"/>
<link rel="alternate" href="https://www.meeds.io/whitepaper" hreflang="en"/>
<link rel="alternate" href="https://www.meeds.io/dapp/fr/livre-blanc" hreflang="fr"/>
<link rel="alternate" href="https://www.meeds.io/dapp/whitepaper" hreflang="en"/>
```

Reply:
done!